db/migrate/20240821114908_change_local_path_type.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeLocalPathType < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured do
```

Comment:
This change is not "safe" in itself, it *will* block the two tables exclusively while changing the data type (it's prolly fast but I can't say for sure).